src/tests/require.ts

```diff
@@ -0,0 +1,7 @@
+import { doNotExecute, Equal, Expect } from "./utils";
+
+doNotExecute(() => {
```

Comment:
Checking - does `require` have any other API's? Like any second arguments?

Reply:
From [the documentation](https://nodejs.org/api/modules.html#requireid) it doesn't seem so. `NodeRequire` extends `NodeJS.Require`, where the latter has a few other attributes but since we're just overwriting `NodeRequire` these aren't affected.